app/services/whatsapp/providers/whatsapp_cloud_service.rb

```diff
@@ -66,14 +69,20 @@ def send_text_message(phone_number, message)
     process_response(response)
   end
 
-  def send_attachment_message(phone_number, message)
-    attachment = message.attachments.first
-    type = %w[image audio video].include?(attachment.file_type) ? attachment.file_type : 'document'
-    type_content = {
-      'link': attachment.download_url
-    }
-    type_content['caption'] = message.content unless %w[audio sticker].include?(type)
+  def process_attachments(phone_number, message)
+    responses = []
+    message.attachments.each do |attachment|
+      responses << send_attachment_message(phone_number, message, attachment)
```

Comment:
Should we wrap this in a `begin`-`rescue` block? Or is it handled somewhere in the above levels?

Also, since we're not using the `responses` in the `send_message` method, does it make sense to return it in the first place?

Reply:
We can rescue the individual attachment-sending functionality. Not doing it here, just to tracking the standard exceptions.